### ISSUE TITLE
Image not hidden on fold in edit mode

### ISSUE THREAD

**Issue by jsmorabito** [ozntel/oz-image-in-editor-obsidian#75]
I'm not sure if another plugin is conflicting with this one. I made a test database and this behavior does not occur but in my main one this is happening. Any ideas? I tried toggle a bunch of plugins on an off with no luck

https://user-images.githubusercontent.com/46250921/132993687-56e98bbe-4bed-4a66-8de4-62977a3cf4b8.mov

**Comment by GitMurf** [comment]
@jsmorabito I agree I wish this could be fixed/resolved but the workaround is to make sure to add an extra line break after the image. Basically the plugin must be adding the image rendering in an HTML element that doesn't get "caught" in the Obsidian native folding event... but if you add a new blank line under it (has to be at the same bullet level if in a list like your example, then the folding knows to go all the way past the image and its rendered element to the next line/bullet to fold it all so the image doesn't stay rendered.

The same thing happens with header / section folding.

I know it is still not ideal but this at least solves the problem for me. See the demo below.

https://user-images.githubusercontent.com/64155612/133148016-a9c5a2d7-a15c-4512-a2c2-cba721a1725e.mp4

**Comment by jsmorabito** [comment]
> @jsmorabito I agree I wish this could be fixed/resolved but the workaround is to make sure to add an extra line break after the image. Basically the plugin must be adding the image rendering in an HTML element that doesn't get "caught" in the Obsidian native folding event... but if you add a new blank line under it (has to be at the same bullet level if in a list like your example, then the folding knows to go all the way past the image and its rendered element to the next line/bullet to fold it all so the image doesn't stay rendered.
> 
> The same thing happens with header / section folding.
> 
> I know it is still not ideal but this at least solves the problem for me. See the demo below.
> 
>  jKNZrgYEkm.mp4

Thanks for the tip @GitMurf ! So is even worth bringing up on the forums/is this something that's possible to fix from a community plugin? it is a pretty niche issue...

**Comment by GitMurf** [comment]
The dev @ozntel will respond here at some point I’m sure.
